setuper/submissions.py

```diff
@@ -25,3 +25,21 @@ def org_unit_gps_point(org_unit):
 
 def submission_org_unit_gps_point(org_unit):
     return f"{org_unit['latitude']} {org_unit['longitude']} {org_unit['altitude']}"
+
+
+def picture_by_org_unit_type_name(org_unit_type_name):
+    picture_name = ""
+    match org_unit_type_name:
```

Comment:
[Structural Pattern Matching](https://docs.python.org/3/whatsnew/3.10.html#pep-634-structural-pattern-matching) was introduced in Python 3.10 but we're still using Python 3.9 on iaso.

I'm afraid this could fail when launched on AWS machines. What do you think?

Reply:
To avoid break something, I think it's better to keep the structural pattern if ... else. I will change it to keep it works with Python 3.9.